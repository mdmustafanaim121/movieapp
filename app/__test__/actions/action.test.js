import getMovies from "../../actions/actions";
import React from 'react';
import renderer from 'react-test-renderer';

describe('Testing Actions', () => {
    it('Should return Action snapshot',()=>{
        expect(getMovies()).toMatchSnapshot()
        })
})
