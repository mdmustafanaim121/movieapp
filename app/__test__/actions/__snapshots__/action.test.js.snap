// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Actions Should return Action snapshot 1`] = `
Promise {
  "_U": 0,
  "_V": 0,
  "_W": null,
  "_X": null,
}
`;
